game-core/src/main/java/games/strategy/triplea/ai/pro/logging/ProLogWindow.java

```diff
@@ -385,18 +369,15 @@ private void settingsDetailsButtonActionPerformed() {
     dialog.setVisible(true);
   }
 
-  void addMessage(final Level level, final String message) {
+  void addMessage(final String message) {
     try {
       if (currentLogTextArea == null) {
         currentLogTextArea = aiOutputLogArea;
       }
       currentLogTextArea.append(message + "\r\n");
     } catch (
         final Exception ex) { // This is bad, but we don't want TripleA crashing because of this...
```

Comment:
Site-Note: I'd agree that this is bad, but I'd really let any exception propagate in this case 🤷